reinforcement_learning/config/train_a2c_policy.yaml

```diff
@@ -0,0 +1,25 @@
+# Participants' configuration
+usersim_config: data/config/usersim_agenda_config.yaml
+agent_config: data/config/moviebot_config_rb_nlu.yaml
+
+# Model's parameters
+model:
+  model_type: a2c
```

Comment:
Can we also call it algorithm for the consistency between config files?

Reply:
Good point! Renamed.